src/Components/Home/Home.css

```diff
@@ -0,0 +1,89 @@
+
+/* showcase */
+.showcase{
+    /* width: 100%; */
+    height: 400px;
+    background: url('../../assets/homeImg.jpg') center center / 100% 100% no-repeat;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+    text-align: center;
+    padding-top: 80px;
+    margin: 1rem 1rem 1rem ;
+}
+.title{
+    text-align: center;
+    margin-top: 30px;
+    color:var(--font--secondary-color);
+    font-size: 40px;
+}
+/* homecards */
+.grid-4-col{
+    margin: 2rem auto;
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    grid-gap:1.5rem;
+    max-width: 58rem;
+
+}
+.home-card{
+    position: relative;
+    /* max-width: 200px; */
```

Comment:
Can remove commented code

Reply:
I may use this for small screen , so for now kept it commented.If not required I will remove it later